scripts/run-artifactory-container.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" > /dev/null && pwd )"
-export ARTIFACTORY_VERSION=${ARTIFACTORY_VERSION:-7.27.10}
+export ARTIFACTORY_VERSION=${ARTIFACTORY_VERSION:-7.35.2}
```

Comment:
We have a ticket to include the version we tested against. Please include this in the read me for now

Reply:
We thought to add it on the top of CHANGELOG.md
The version here is not related to that ticket.